### DISCUSSION TITLE
오픈 프로필에 하트 수를 가져오는 방법

### DISCUSSION THREAD

**Discussion by ghost** [storycraft/node-kakao#316] [General]
카카오톡의 오픈채팅 프로필의 하트수를  가져오려면 어케해요?

**Comment by raravel** [comment]
#310 에서 검색한 결과 중 rc 라는 값이 하트 수인데, 특정 링크 아이디로 가져오는 것은 찾지 못 하겠습니다.

**Comment by storycraft** [comment]
TalkOpenChannelList 객체의 getReaction 메서드를 통해 하트 갯수와 여부를 가져올수 있습니다.
https://github.com/storycraft/node-kakao/blob/9965d5640a2d92e60f962196c5584952d13278c5/src/talk/openlink/talk-open-channel-list.ts#L256
